scio-bigquery/src/test/scala-2.11/com/spotify/scio/bigquery/types/TypeProviderTestScala211.scala

```diff
@@ -64,4 +64,11 @@ class TypeProviderTestScala211 extends FlatSpec with Matchers {
   it should "not provide .tupled in companion object with more than 22 fields" in {
     ArtisanalMoreThan22Fields.getClass.getMethods.map(_.getName) should not contain "tupled"
   }
+  
+  // This should compile
+  @BigQueryType.toTable
```

Comment:
Would it make sense to add some tests on the `TwentyThree` type - similar to the `ArtisanalMoreThan22Fields` tests?